packages/os/hachiko.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description= Waits for host to be in-service before running kubelet if autoscaling.should_wait is true.
+After=network-online.target settings-applier.service
+Before=kubelet.service
+Requires=settings-applier.service network-online.target
+# We only want to run once, at first boot. This file is created by hachiko
+# after a successful run.
+ConditionPathExists=!/var/lib/bottlerocket/hachiko.ran
+
+[Service]
```

Comment:
I think we probably want `RemainAfterExit=true` for this oneshot service so systemd doesn't show it as "deactivating" or "dead" after it runs.

See the service unit for `early-boot-config` for example.